ui/r4r/src/main/kotlin/de/cyface/app/r4r/capturing/CapturingFragment.kt

```diff
@@ -794,6 +794,7 @@ class CapturingFragment : Fragment(), DataCapturingListener, CameraListener {
             true, // For testing, switch this to false to capture w/1 Hz
             triggeringDistance,
             false,
+            1,
```

Comment:
Is this on purpose that RFR uses "1" which is V8NAnonymization, see `BackgroundService` ?

If this is on purpose, then **don't merge this** until Model 2 works.

Reply:
On the Main branch I've selected `0` (Legacy model) for now, until the new model does not crash anymore.